src/applications/simple-forms-form-engine/shared/utils/formConfig.js

```diff
@@ -45,21 +45,33 @@ const formatChapters = chapters =>
     {},
   );
 
+export const statementOfTruthBody =
+  'I confirm that the identifying information in this form is accurate and ' +
+  'has been represented correctly.';
+
+/** @returns {FormConfig} */
 export const createFormConfig = (form, options) => {
   const { chapters, formId, ombInfo, title } = form;
   const { rootUrl, trackingPrefix } = options;
   const subTitle = `VA Form ${formId}`;
 
   return {
+    preSubmitInfo: {
+      statementOfTruth: {
+        body: statementOfTruthBody,
+        messageAriaDescribedby: statementOfTruthBody,
+        fullNamePath: 'fullName',
```

Comment:
This makes me wonder what happens if there isn't a personal-info page for any reason. This shouldn't be the case, I recognize, but this necessary connection raises some curiosity for me. This is not a blocking concern at all, but I'm curious - do we know what happens if the provided path is not a path that exists on the form?

Reply:
Great question. I tested this scenario out by removing the "Your Personal Information" chapter from the mock 21-4140 form and filling out the form locally. The form requires no data in the "Your full name" field and gets upset if you put a name (any name) there.

This is probably not desirable behavior, but the error here lies in a missing YPI chapter, not with the statement of truth. We should probably not render any form that is missing that chapter. 

- I've created department-of-veterans-affairs/va.gov-team#98440 to address this issue. It needs input from Product and Design on how we want the renderer to behave in this situation.